monitoring/index.adoc

```diff
@@ -0,0 +1,28 @@
+[id="monitoring-overview"]
+= Monitoring overview
+include::modules/common-attributes.adoc[]
+:context: monitoring-overview
+
+toc::[]
+
+The growth of containerization has not only brought many advantages to developers,DevOps, and IT teams but also challenges. The flexibility and scalability of containerized applications pose challenges since there is no direct relationship between an application and the server it runs on.
+A cluster administrator can use the pre-installed, pre-configured and self updating {product-title} xref:../monitoring/understanding-the-monitoring-stack.adoc#understanding-the-monitoring-stack[monitoring stack] to monitor:
+
+* Default core platform components
+* (Optional) components for user-defined projects
+
+The monitoring stack is available by default, when {product-title} is installed. The cluster administrator has the option to enable xref:../monitoring/enabling-monitoring-for-user-defined-projects.adoc#enabling-monitoring-for-user-defined-projects[monitoring for user-defined projects].
+You can also xref:../monitoring/configuring-the-monitoring-stack.adoc#configuring-the-monitoring-stack[configure the monitoring stack] with the supported configurations.
+You can use the **Administrator** or ** Developer** perspective in the {product-title} monitoring dashboard to:
+
+* xref:../monitoring/managing-metrics.adoc#managing-metrics[Manage metrics]
+* xref:../monitoring/managing-alerts.adoc#managing-alerts[Manage alerts]
+* xref:../monitoring/reviewing-monitoring-dashboards.adoc#reviewing-monitoring-dashboards[Review monitoring dashboards]
+
+The {product-title} web console provides xref:../monitoring/accessing-third-party-uis.adoc#accessing-third-party-uis[third-party integrated UIs] such as metrics,alerts, and review of monitoring dashboards. You can access these third-party user interfaces (UIs) either through the web console or the CLI.
+You can also xref:../monitoring/exposing-custom-application-metrics-for-autoscaling.adoc#exposing-custom-application-metrics-for-autoscaling[expose custom application metrics] for horizontal pod autoscaling.
+
+As a cluster administrator you can determine why:
+
+* xref:../monitoring/troubleshooting-monitoring-issues.adoc#troubleshooting-monitoring-issues[User-metrics are not available]
```

Comment:
It is better change the link to
`../monitoring/troubleshooting-monitoring-issues.adoc#troubleshooting-monitoring-issues#investigating-why-user-defined-metrics-are-unavailable_troubleshooting-monitoring-issues`